pkgs/development/tools/parsing/tree-sitter/default.nix

```diff
@@ -26,15 +27,20 @@ let
     inherit writeShellScript nix-prefetch-git curl jq xe src;
   };
 
+  fetch = (v: fetchgit {inherit (v) url rev sha256 fetchSubmodules; });
+
   grammars =
-    let fetch =
-      (v: fetchgit {inherit (v) url rev sha256 fetchSubmodules; });
-    in runCommand "grammars" {} (''
+    runCommand "grammars" {} (''
        mkdir $out
      '' + (lib.concatStrings (lib.mapAttrsToList
             (name: grammar: "ln -s ${fetch grammar} $out/${name}\n")
             (import ./grammars))));
 
+  builtGrammars = let
+    change = name: grammar:
+      callPackage ./library.nix { language = name; version = "placeholder"; source = fetch grammar;}
+    ;
+    in lib.mapAttrs change (removeAttrs (import ./grammars) ["typescript"]);
```

Comment:
Can you add some reasoning here? What is the error?

Reply:
`error: /nix/store/p4azyipnwcq6sv33wl0vd5b5xm5qfrir-tree-sitter-typescript-aa950f5/src/parser.c: No such file or directory`
I dont feel like making a special case for typescrip.

Reply:
Let’s add that as a comment

Reply:
added a ;ention